core/src/main/resources/hudson/views/GlobalDefaultViewConfiguration/config.jelly

```diff
@@ -0,0 +1,36 @@
+<!--
+The MIT License
+Copyright (c) 2025, Markus Winter
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <j:if test="${app.views.size() > 0}">
+    <f:entry title="${%Default View}" field="defaultView">
```

Comment:
```suggestion
    <f:entry title="${%Default view}" field="defaultView">
```
Although based on https://github.com/search?q=repo%3Ajenkinsci%2Fjenkins+%22default%5C%5C+view%22+language%3A%22Java+Properties%22&type=code it would seem to be case-insensitive.

Reply:
It is case sensitive. I think I still had the config.groovy around somewhere when testing that part.
Your search shows that we're inconsistent, in the user preferences we use `Default View` instead. And the resource bundle for GlobalSecurityConfiguration/config seems to be obsolete the is no config.jelly or groovy